x-pack/plugins/lens/public/datatable_visualization/components/table_basic.scss

```diff
@@ -11,3 +16,7 @@
 .lnsDataTable__filter:focus-within {
   opacity: 1;
 }
```

Comment:
Is this selector being used still? I didn't see it in my testing. If not, should probably be removed.